flake-module.nix

```diff
@@ -114,6 +114,20 @@ in
                     description = ''Package overrides given new source path'';
                     default = { };
                   };
+                  easy-overrides =
+                    mkOption {
+                      description = " An attribute set with haskell packages and the options.";
+                      type = with types; functionTo (functionTo (attrsOf (submodule {
```

Comment:
@roberth can you help here if there is a better way to implement this type.

Reply:
See thread https://github.com/srid/haskell-flake/pull/87#discussion_r1113327459